luarules/gadgets/game_custom_map_tidal.lua

```diff
@@ -0,0 +1,25 @@
+function gadget:GetInfo()
+	return {
+		name = "Custom Map Tidal",
+		desc = "Sets map tidal for modoption via engine call",
+		author = "",
+		date = "",
+		license = "Horses",
```

Comment:
oi m8, you got a loicence for that wupget?

Reply:
No joke licenses, they've caused us enough headache already